addons/l10n_ro_edi_stock/models/stock_picking.py

```diff
@@ -0,0 +1,864 @@
+    ('362903', "BVF Tulcea (ROGL8900)"),
+    ('362904', "BVF Isaccea (ROGL8920)"),
+    ('372801', "BVI Vaslui (ROIS9610)"),
+    ('372901', "BVF Fălciu (-)"),
+    ('372902', "BVF Albiţa (ROIS0100)"),
+    ('382801', "BVI Râmnicu Vâlcea (ROCR7700)"),
+    ('392801', "BVI Focșani (ROGL3600)"),
+    ('402801', "BVI Bucureşti Poştă (ROBU1380)"),
+    ('402802', "BVI Târguri și Expoziții (ROBU1400)"),
+    ('402901', "BVF Băneasa (ROBU1040)"),
+    ('512801', "BVI Călăraşi (ROCT1710)"),
+    ('522801', "BVI Giurgiu (ROBU3910)"),
+    ('522901', "BVF Zona Liberă Giurgiu (ROBU3980)"),
+]
+
+STATE_CODES = {
+    'AB': '1',
+    'AR': '2',
+    'AG': '3',
+    'BC': '4',
+    'BH': '5',
+    'BN': '6',
+    'BT': '7',
+    'BV': '8',
+    'BR': '9',
+    'BZ': '10',
+    'CS': '11',
+    'CJ': '12',
+    'CT': '13',
+    'CV': '14',
+    'DB': '15',
+    'DJ': '16',
+    'GL': '17',
+    'GJ': '18',
+    'HR': '19',
+    'HD': '20',
+    'IL': '21',
+    'IS': '22',
+    'IF': '23',
+    'MM': '24',
+    'MH': '25',
+    'MS': '26',
+    'NT': '27',
+    'OT': '28',
+    'PH': '29',
+    'SM': '30',
+    'SJ': '31',
+    'SB': '32',
+    'SV': '33',
+    'TR': '34',
+    'TM': '35',
+    'TL': '36',
+    'VS': '37',
+    'VL': '38',
+    'VN': '39',
+    'B': '40',
+    'CL': '51',
+    'GR': '52',
+}
+
+
+class Picking(models.Model):
+    _inherit = 'stock.picking'
+
+    # Document fields
+    l10n_ro_edi_stock_document_ids = fields.One2many(comodel_name='l10n_ro_edi.document', inverse_name='picking_id')
+    l10n_ro_edi_stock_document_message = fields.Char(compute='_compute_l10n_ro_edi_stock_current_document_data')
+    l10n_ro_edi_stock_document_load_id = fields.Char(compute='_compute_l10n_ro_edi_stock_current_document_data', string="eTransport Id")
+    l10n_ro_edi_stock_document_uit = fields.Char(compute='_compute_l10n_ro_edi_stock_current_document_data', string="eTransport UIT")
+    l10n_ro_edi_stock_state = fields.Selection(selection=DOCUMENT_STATES, compute='_compute_l10n_ro_edi_stock_current_document_data', string="eTransport Status")
+
+    # Data fields
+    l10n_ro_edi_stock_operation_type = fields.Selection(selection=OPERATION_TYPES, string="Operation Type")
+    l10n_ro_edi_stock_available_operation_scopes = fields.Char(compute='_compute_l10n_ro_edi_stock_available_operation_scopes')
+    l10n_ro_edi_stock_operation_scope = fields.Selection(selection=OPERATION_SCOPES, string="Operation Scope")
+
+    l10n_ro_edi_stock_vehicle_number = fields.Char(string="Vehicle Number", size=20)
+    l10n_ro_edi_stock_trailer_1_number = fields.Char(string="Trailer 1 Number", size=20)
+    l10n_ro_edi_stock_trailer_2_number = fields.Char(string="Trailer 2 Number", size=20)
+
+    l10n_ro_edi_stock_available_start_loc_types = fields.Char(compute='_compute_l10n_ro_edi_stock_available_location_types')
+    l10n_ro_edi_stock_start_loc_type = fields.Selection(selection=LOCATION_TYPES, string="Location Type", default='location')
```

Comment:
Don't put any default on localisation modules. It has to be computed fields that assign a value only when the record has the related country.